### PR TITLE
Server Migration Documentation Fix

### PR DESCRIPTION
**Pull request by mmattel**
This is a fix for issue #803 (Server Migration Documentation Fix)
- add `apps` directory in rsync
- added some text
- improved some text

@pclemins fyi

### CODE REVIEW

File: modules/admin_manual/pages/maintenance/migrating.adoc [hunk 1]
```diff
@@ -143,7 +142,7 @@ cd /var/www/owncloud/
 sudo -u www-data php occ maintenance:mode --on
 ----
 
-After that’s done, wait for 6-7 minutes and stop Apache:
+After that’s done, wait for some minutes and stop your web server, in this case Apache:
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/migrating.adoc]: s/wait for some minutes/wat a few minutes/g

File: modules/admin_manual/pages/maintenance/migrating.adoc [hunk 2]
```diff
@@ -242,6 +247,8 @@ https://certbot.eff.org/[certbot].
 Now you need to reverse the change to ssh-config. Specifically, set `PermitRootLogin`
 to `no` and restart ssh. To do that, run the following command:
 
+NOTE: This is a security measure and improves ssh security.
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/migrating.adoc]: s/ssh/SSH/g

File: modules/admin_manual/pages/maintenance/migrating.adoc (hunk 1 above)
Comment (phil-davis) [reply modules/admin_manual/pages/maintenance/migrating.adoc]: then 
s/wat/wait/
;)

File: modules/admin_manual/pages/maintenance/migrating.adoc [hunk 3]
```diff
@@ -93,18 +93,17 @@ long as it sends the correct `X-Forwarded-Host` header.
...
-Now, let’s step through an example migration. For this example to work,
-you will need the following on both the servers that you will use for
-the migration:
+The following is an example migration with assumptions to make
+this migration work:
 
-* Ubuntu 16.04
+* Ubuntu 16.04+
 * SSH with `PermitRootLogin` set to `yes`
+* Database used is MySql / MariaDB
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/migrating.adoc]: s/MySql/MySQL/g

**Pull request merged**